### DISCUSSION TITLE
Other cryptocurrencies? Implement nano (XNO) currency for peer-to-peer payments (or others)

### DISCUSSION THREAD

**Discussion by vinibarbosabr** [OpenBazaar/openbazaar3-rust#2] [Ideas]
I wonder if there are any plans to allow other cryptocurrencies and decentralized peer-to-peer networks for payments in the marketplace.

This would give sellers and buyers **more options in sending and receiving peer-to-peer payments**, allowing them to choose and close a deal **in their currency of preference**, competing freely with their own advantages and disadvantages in a real free market.

That said. I wonder if any implementation development from the open source **nano community** would be welcome in this regard. Or if it would be prevented in any way by the project managers.

I might be able to start a bounty to incentive this implementation by people with the proper skill set, as I'm not a developer my self. _I'm just a huge enthusiast of using decentralized peer-to-peer money (and I actually receive ~20% of my monthly income in bitcoin, nano, bch and monero — so it would be interesting being able to spend them directly in OpenBazaar)._

Talking exclusively about **XNO** (aka nano), I also think that OpenBazaar could directly benefit by allowing its implementation.

XNO has straight forward characteristics that make it a **great, secure, and very efficient medium of exchange** — allowing buyers to pay less and get quicker confirmation that their purchase was successful; while allowing seller to receive more with the certainty of deterministic finality settlement of irreversible monetary transactions.

This happens because nano operates in a feeless network (no fees for the end user), and ~0.3sec network confirmations for each p2p transactions made.

**More information can be found in:** https://docs.nano.org/ and https://docs.nano.org/integration-guides/the-basics/

**You can also check nano-node repo:** https://github.com/nanocurrency/nano-node
or **the under-development implementation in Rust:** https://github.com/simpago/rsnano-node

**Comment by nano2dev** [comment]
I'd like to add to Vini's comment.

The best implementations are always non-custodial.

Users get a Nano Private Key, which they use to [sign](https://github.com/numsu/nanocurrency-web-js) Nano blocks client-side. Blocks can then be sent off to any Public RPC that supports "process" calls. 

I'd like to contribute Nano.to's Public RPC to this cause. 

https://rpc.nano.to

Using a Public RPC allows anyone to set up a Nano implementation MVP in minutes ;)
